tests/phpunit/tests/taxonomy.php

```diff
@@ -125,7 +125,12 @@ public function test_taxonomy_exists_unknown() {
 		$this->assertFalse( taxonomy_exists( rand_str() ) );
 		$this->assertFalse( taxonomy_exists( '' ) );
 		$this->assertFalse( taxonomy_exists( 0 ) );
-		$this->assertFalse( taxonomy_exists( null ) );
+		$this->assertFalse( taxonomy_exists( 1 ), 'Did not return false with (int) 1' );
```

Comment:
```suggestion
	}

	public function test_taxonomy_exists_invalid_parameter_type()
		$this->assertFalse( taxonomy_exists( 1 ), 'Did not return false with (int) 1' );
```

Nitpick but they seem subtly different tests.